dkg/pedersen/handler.go

```diff
@@ -513,5 +656,321 @@ func (h *Handler) handleDeal(ctx context.Context, msg types.Deal, out mino.Sende
 		}
 	}
 
+	dela.Logger.Debug().Msgf("%s is certified", h.me)
+
 	return nil
 }
+
+func (h *Handler) announceDkgPublicKey(isCommonNode bool, out mino.Sender, from mino.Address) error {
+	// 6. Send back the public DKG key
+
+	var publicKey kyber.Point
+
+	isOldNode := h.startRes.GetDistKey() != nil
+	isNewNode := !isOldNode
```

Comment:
```suggestion
	isNewNode := h.startRes.GetDistKey() == nil
```
old node doesn't seem used in this function.